metals/src/main/scala/scala/meta/internal/metals/Icons.scala

```diff
@@ -65,7 +65,7 @@ object Icons {
     override def check: String = "$(check) "
     override def findsuper: String = "$(arrow-up)"
     override def folder: String = "$(folder)"
```

Comment:
Should we also do it for each icon used in the NewProjectProvider? Like for `folder` ?

Reply:
When testing it locally the only one I noticed that seemed not aligned was the github icon. I just checked and changing the folder one doesn't seem to affect anything because it seems that we aren't adding an extra space for it when it's used. As far as I can tell, only the GitHub one needs to change here.